config/locales/en.yml

```diff
@@ -2,77 +2,97 @@ en:
  app_name: 'if me'
  app_description: 'a community for mental health experiences'
  email: 'join.ifme@gmail.com'
- github: 'GitHub'
- twitter: 'Twitter'
- facebook: 'Facebook'
  us: 'us'
- patreon: 'Patreon'
- slack: 'Slack'
- linebreak: 'Linebreak'
- feedback: 'feedback'
- help: 'help'
- edit: 'Edit'
- new: 'New'
- category: 'Category'
- medication: 'Medication'
- meeting: 'Meeting'
- moment: 'Moment'
- mood: 'Mood'
- group: 'Group'
- strategy: 'Strategy'
- add: 'Add'
- add_all: 'Add all'
- select_all: 'Select all'
- cancel: 'Cancel'
- accept: 'Accept'
- reject: 'Reject'
- remove: 'Remove'
- delete: 'Delete'
  ellipsis: " [...]"
- view_more: 'View more'
- stories: 'Stories'
- welcome: 'Welcome'
  created: '<strong>Created:</strong> %{created_at}'
  edited: '<strong>Created:</strong> %{created_at} <em>(edited)</em>'
  edited_updated_at: '<strong>Created:</strong> %{created_at} <em>(edited %{updated_at})</em>'
- daily_reminder: 'Daily reminder email'
  salutation: "Hi %{name},"
  click_here: "click here"
 
+ support:
```

Comment:
nope! just didn't catch that it defaults to English when i was reading up on it. will remove.